src/Hosting/Hosting/src/GenericHost/GenericWebHostBuilder.cs

```diff
@@ -86,6 +86,7 @@ public GenericWebHostBuilder(IHostBuilder builder, WebHostBuilderOptions options
                 // We need to flow this differently
                 services.TryAddSingleton(sp => new DiagnosticListener("Microsoft.AspNetCore"));
                 services.TryAddSingleton<DiagnosticSource>(sp => sp.GetRequiredService<DiagnosticListener>());
+                services.TryAddSingleton(sp => new ActivitySource("Microsoft.AspNetCore"));
```

Comment:
Why did the value change from "Microsoft.AspNetCore.Hosting"?

Reply:
Matches the diagnostic listener. @shirhatti are they used in a similar way?

Reply:
This is technically a breaking change. You've changed the ActivitySource name. Most subscribers only care about the Activity name, but I'd prefer not to have this change

Reply:
Didn't we make this change in 6.0?

Reply:
The original ActivitySource guidance was to have something like ActivitySource Name = "Microsoft.AspNetCore.Hosting" and Activity Name = "BeginRequest", that is don't use a fully qualified name for the Activity. However, it wasn't possible to do that without breaking existing consumers, so we just changed the guidance for Activity 😄

Reply:
Yes, new in 6.0

Reply:
Then it's not a breaking change. We just need the "right" naming.